data/json/recipes/recipe_weapon.json

```diff
@@ -1922,6 +1949,29 @@
     "tools": [  ],
     "components": [ [ [ "glass_shard", 1 ] ], [ [ "rag", 1 ], [ "duct_tape", 10 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "bone_knife",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_PIERCING",
+    "skill_used": "fabrication",
+    "skill_required": [ "survival", 1 ],
+    "time": "2 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ "bone", 1 ], "bone_human", 1 ]
```

Comment:
```suggestion
    "components": [ [ [ "bone", 1 ], [ "bone_human", 1 ] ] ]
```

Reply:
thanks, caught up to it after my day shift.